README.md

```diff
@@ -136,3 +136,56 @@ If you do not have Python 3.10 installed, you can use root-less podman with a Py
 ```bash
 podman run --rm --tty --volume "$(pwd):/mnt:z" --workdir /mnt docker.io/library/python:3.10 bash -c 'pip install nox ; nox -s pip-compile'
 ```
+
+## Creating release tags
+
+When a tag is created in the [`ansible/ansible`](https://github.com/ansible/ansible) repository for a release or release candidate, a corresponding tag should be created in this `ansible-documentation` repository.
+
+First, install the additional tagging dependencies as follows, creating or activating a `venv` as needed:
+
+```bash
+python3 -m venv ./venv
+source ./venv/bin/activate
+pip insatll -r hacking/tagger/requirements.txt
+```
+
+Next, ensure that you have both [`ansible/ansible`](https://github.com/ansible/ansible) and [`ansible/ansible-documentation`](https://github.com/ansible/ansible-documentation) repositories checked out.  The tool assumes that both checkouts are done from the same directory, or you can override this with the `--docs` and `--remote` options. For example:
+
+```bash
+# Enter your projects directory
+cd ~projects/
+
+# Clone core
+git clone https://github.com/ansible/ansible.git
+
+# Clone docs
+git clone https://github.com/ansible/ansible-documentation.git
+
+# confirm
+ls -1
+ansible
+ansible-documentation
```

Comment:
This seems a little too much detail. I think it's better to be concise and just make it clear that both repos need to be checked out at the same level of the filesystem.

Reply:
I pulled it back out. Thanks for the feedback